bin/cyhy-import

```diff
@@ -22,15 +22,17 @@ Notes:
 import datetime
 import json
 import sys
+from collections import defaultdict
 
 # Third-Party Libraries
 import dateutil
 from docopt import docopt
 import progressbar as pb
-import netaddr
+from netaddr import IPSet
```

Comment:
This should be above the `progressbar` import.

Reply:
implementing import of IPSet over all of netaddr